index.php

```diff
@@ -1,18 +1,19 @@
 <?php
 require_once __DIR__ . '/vendor/autoload.php';
 
-$postGlobal = $_POST;
 $db = new \TopDog\Classes\PDOConnection();
 $dbHandler = new \TopDog\Classes\DbHandler($db);
 $dropdownMaker = new \TopDog\Classes\DropdownMaker();
-$formHandler = new \TopDog\Classes\FormHandler($postGlobal);
 $dogDisplayer = new \TopDog\Classes\DogDisplayer();
-$dogManager = new \TopDog\Classes\DogManager($dbHandler, $dropdownMaker, $formHandler, $dogDisplayer);
+$dogManager = new \TopDog\Classes\DogManager($dbHandler, $dropdownMaker, $dogDisplayer);
 
 if(isset($_POST["Breeds"])) {
-    $dogManager->formGetId();
-    $dogManager->populateDogs();
-    $dogManager->getFaveId();
+    if (isset($_POST['favDogId'])) {
+        $dogManager->faveToDb($_POST['favDogId'], $_POST['Breeds']);
+    }
+    $breeds = $_POST['Breeds'];
```

Comment:
You might want to set this to a variable at the top so we can re-use it in the if statement as well